[2025-03-05T04:50:12.038+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-05T04:50:12.078+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: forecast_stock_prices.store_forecast_in_snowflake manual__2025-03-05T04:49:20.785675+00:00 [queued]>
[2025-03-05T04:50:12.096+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: forecast_stock_prices.store_forecast_in_snowflake manual__2025-03-05T04:49:20.785675+00:00 [queued]>
[2025-03-05T04:50:12.097+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-03-05T04:50:12.126+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): store_forecast_in_snowflake> on 2025-03-05 04:49:20.785675+00:00
[2025-03-05T04:50:12.151+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=839) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-05T04:50:12.150+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'forecast_stock_prices', 'store_forecast_in_snowflake', 'manual__2025-03-05T04:49:20.785675+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/forecast_stock_prices.py', '--cfg-path', '/tmp/tmpo90cc2fm']
[2025-03-05T04:50:12.154+0000] {standard_task_runner.py:72} INFO - Started process 856 to run task
[2025-03-05T04:50:12.154+0000] {standard_task_runner.py:105} INFO - Job 14: Subtask store_forecast_in_snowflake
[2025-03-05T04:50:12.240+0000] {task_command.py:467} INFO - Running <TaskInstance: forecast_stock_prices.store_forecast_in_snowflake manual__2025-03-05T04:49:20.785675+00:00 [running]> on host 0d515916445a
[2025-03-05T04:50:12.426+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='forecast_stock_prices' AIRFLOW_CTX_TASK_ID='store_forecast_in_snowflake' AIRFLOW_CTX_EXECUTION_DATE='2025-03-05T04:49:20.785675+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-05T04:49:20.785675+00:00'
[2025-03-05T04:50:12.429+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-05T04:50:12.450+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-05T04:50:12.452+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-5.15.167.4-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2025-03-05T04:50:12.455+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-05T04:50:12.457+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-05T04:50:13.098+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-05T04:50:13.710+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-05T04:50:13.712+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-05T04:50:13.729+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-05T04:50:13.733+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=forecast_stock_prices, task_id=store_forecast_in_snowflake, run_id=manual__2025-03-05T04:49:20.785675+00:00, execution_date=20250305T044920, start_date=20250305T045012, end_date=20250305T045013
[2025-03-05T04:50:13.817+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-05T04:50:13.904+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-05T04:50:13.917+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
